repl.lua

```diff
@@ -655,17 +633,36 @@ local bindings = {
 	{ 'meta+v',      function() paste(true) end             },
 	{ 'ctrl+w',      del_word                               },
 }
--- Add bindings for all the printable US-ASCII characters from ' ' to '~'
--- inclusive. Note, this is a pretty hacky way to do text input. mpv's input
--- system was designed for single-key key bindings rather than text input, so
--- things like dead-keys and non-ASCII input won't work. This is probably okay
--- though, since all mpv's commands and properties can be represented in ASCII.
-for b = (' '):byte(), ('~'):byte() do
-	local c = string.char(b)
-	local binding = binding_name_map[c] or c
-	bindings[#bindings + 1] = {binding, function() handle_char_input(c) end}
-end
-add_repl_bindings(bindings)
+
+local function text_input(info)
+	if info.key_text and (info.event == "press" or info.event == "down"
+						  or info.event == "repeat")
```

Comment:
I tend to use tabs for indentation and spaces for alignment in my personal projects, so this line should start with one tab and have the rest as spaces.

Though, if this script is going to become part of mpv, I'll probably reindent it with spaces at some point to match the rest of mpv.

Reply:
Yeah, it should be de-tabbed when put into mpv master. This specific line aligns for me with a tab width of 4.